### ISSUE TITLE
Lemuroid cannot search in folders

### ISSUE THREAD

**Issue by esclusoilcane** [Swordfish90/Lemuroid#342]
I used the good lemuroid in different systems without so much problem, when I tried to install it on gt king 922 android box, it doesn't allow me to search any file. I mean, i click on directory but nothing happens. I tried to give permissions manually again and again, i also tried many version of lemuroid, but the same. How can solve? Thanks!!

**Comment by sahiluchil** [comment]
I have a Xiaomi Redmi Note 5 Pro with custom ROM Havoc-OS Android 10 running on it. I was also getting the same issue. Used a screen recorder to get what the issue was.
I was getting this message.
"THIS DEVICE DOES NOT SUPPORT THE STORAGE ACCESS FRAMEWORK FILE PICKER. PLEASE MANUALLY COPY UNCOMPRESSED ROM FILES INTO /storage/emulated/0/Android/data.com.swordfish.lemuroid/files/roms AND HIT RESCAN"

![Lemuroid Photo](https://user-images.githubusercontent.com/62016455/147402768-49164341-f127-4193-9c2d-fef85d18b67e.jpg)

**Comment by esclusoilcane** [comment]
Yes probably u have to put roms in default folder, In my case I gave my gt king slimbox rom and everything became faster and functional, even lemur Droid
